packages/contact-manager/src/agent/ContactManager.ts

```diff
@@ -176,6 +177,9 @@ export class ContactManager implements IAgentPlugin {
   /** {@inheritDoc IContactManager.cmAddIdentity} */
   private async cmAddIdentity(args: AddIdentityArgs, context: RequiredContext): Promise<Identity> {
     const { contactId, identity } = args
+    if (!identity.origin) {
```

Comment:
I removed these lines. In the entity we have
>   @Column('simple-enum', { name: 'identity_origin', enum: IdentityOrigin, **nullable: false** })
  origin!: IdentityOrigin

I did add 
> NOT NULL DEFAULT 'EXTERNAL'`
to the migrations though, otherwise we could never add this to a table with existing record.

Reply:
you can because you can write a query that adds a value to existing ones

Reply:
Removed DEFAULT 'EXTERNAL it for PartyType because there is only a CREATE TABLE for that
For identity I changed to to 
```
    await queryRunner.query(`ALTER TABLE "Identity" ADD COLUMN "identity_origin" "public"."IdentityOrigin_type_enum" DEFAULT 'EXTERNAL' NOT NULL`)
    await queryRunner.query(`ALTER TABLE "Identity" ALTER COLUMN "identity_origin" DROP DEFAULT`)
```

Reply:
PartyType still has a default in the entity. While the migrations do not have a default on PartyType. so in the entity it should be removed.

This is reversed for the indentity_origin. the entity does not have one but the migrations have one.

Reply:
> PartyType still has a default in the entity
Fixed

Reply:
Next try...